src/dom/DomEvent.DoubleTap.js

```diff
@@ -38,6 +36,8 @@ L.extend(L.DomEvent, {
 		function onTouchEnd() {
 			if (doubleTap && !touch.cancelBubble) {
 				if (L.Browser.pointer) {
+					if (e.pointerType === 'mouse') { return; }

```

Comment:
This is why #5180 was firing `dblclick`s on mouse after a touchscreen double-tap: the `doubleTap` flag is set after a double-tap.